### ISSUE TITLE
Consider making Post Template block a bundled template part with new semantic area

### ISSUE THREAD

**Issue by jameskoster** [WordPress/gutenberg#44362]
It's common for archive templates like Index, Archive, Category, Tag to display post content the same way. At the moment, if you decide to change the design in one template, you then have to replicate that change across all the other templates as well. Traditionally themes would include a `content.php` template part to circumvent this.

If you try to do this in the block editor, the experience is a little funky (https://github.com/WordPress/gutenberg/issues/44160). 

An alternative solution would be to make the Post Template block a bundled template part in its own right, with a new semantic area (similar to header / footer). A benefit with this approach is that it immediately becomes possible for users to edit in focus mode, swap with other post-content template parts, browse patterns, etc.

cc @WordPress/theme-team for thoughts.

**Comment by jasmussen** [comment]
As a site editor equivalent to content.php, it should be possible to edit this in isolation, in focus mode. That way you can focus just on one generic post where one can manipulate things without being confused by all the content.

As a way to address this, could we leverage the fact that "Post Template" inside a query loop already mostly functions like this, and simply add an "Edit" button to its toolbar?

<img width="368" alt="Screenshot 2022-09-28 at 14 35 35" src="https://user-images.githubusercontent.com/1204802/192780304-542499d6-cba2-429f-883f-ec1fa6f1a6bc.png">

**Comment by jameskoster** [comment]
We could, that would be a nice enhancement. But it only satisfies one half of this issue. 

If it is not made clear that it is synced, it might be too easy to confuse editing in isolation as editing _a single post_.

**Comment by youknowriad** [comment]
My hunch is that this could create more issues than it actually solves. I'am also wondering what are your current thoughts on the subject.

**Comment by jameskoster** [comment]
Yes it's probably too late to make this change now. Theme authors are likely placing generic template parts or synced patterns inside the Post Template block. That's what I do anyway.
